{"uuid":"5c563d11-3c1c-4a0b-b5d5-381e7e810760","historyId":"36030a09f9b1923da241513131a6c206","fullName":"tests.ErrorValidationTest.testInvalidLoginLandingPage","labels":[{"name":"package","value":"tests.ErrorValidationTest"},{"name":"testClass","value":"tests.ErrorValidationTest"},{"name":"testMethod","value":"testInvalidLoginLandingPage"},{"name":"parentSuite","value":"RegressionTest"},{"name":"suite","value":"LoginErrorValidation"},{"name":"subSuite","value":"tests.ErrorValidationTest"},{"name":"host","value":"Qcentris-396"},{"name":"thread","value":"24560@Qcentris-396.TestNG-tests-2(37)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"Validates error message for invalid login attempts on the landing page.","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"java.lang.AssertionError","trace":"java.lang.AssertionError\r\n\tat org.junit.Assert.fail(Assert.java:87)\r\n\tat org.junit.Assert.assertTrue(Assert.java:42)\r\n\tat org.junit.Assert.assertTrue(Assert.java:53)\r\n\tat tests.ErrorValidationTest.testInvalidLoginLandingPage(ErrorValidationTest.java:60)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)\r\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)\r\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)\r\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1596)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:794)\r\n\tat org.testng.TestRunner.run(TestRunner.java:596)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:377)\r\n\tat org.testng.SuiteRunner.access$000(SuiteRunner.java:28)\r\n\tat org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)\r\n\tat org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)\r\n\tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\r\n\tat java.base/java.lang.Thread.run(Thread.java:1583)\r\n"},"stage":"finished","description":"This test performs an invalid login and validates the error message displayed on the landing page.","steps":[],"attachments":[{"name":"error description","source":"258f3bb5-774e-488d-a9c1-55821778e518-attachment.txt","type":"text/plain"},{"name":"Invalid Login Error Screenshot","source":"6c3e7e86-fd2a-47da-9d4c-9ee727333568-attachment"}],"parameters":[],"start":1736346380401,"stop":1736346380865}